modules/cfe_testcase/src/cfe_test.c

```diff
@@ -59,6 +59,7 @@ void CFE_TestMain(void)
     FSHeaderTestSetup();
     SBPipeMangSetup();
     TimeCurrentTestSetup();
+    TimeArithmeticTestSetup();
```

Comment:
Minor thing, I would put arithmetic above current, moving forward I am trying to run the test alphabetically just to have some order.

Reply:
:+1: done